src/Store.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { createContext, useState } from 'react';
+import { Todo } from './types/Todo';
+
+export enum Status {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+export const FilterContext = createContext({
+  filter: Status.All,
+  setFilter: (_value: Status) => {},
+});
+
+export const SelectedTodoContext = createContext({
+  selectedTodo: null as Todo | null,
```

Comment:
```suggestion
  selectedTodo: Todo | null,
```

Reply:
Todo is a type, we can't assign it to the object property as a value